src/Hosting/Abstractions/src/MetricsPrototype/DefaultMeterFactory.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Metrics;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Hosting.Abstractions.MetricsPrototype;
+
+internal sealed class DefaultMeterFactory : IMeterFactory
+{
+    private readonly IOptions<MeterFactoryOptions> _options;
+
+    public DefaultMeterFactory(IOptions<MeterFactoryOptions> options)
+    {
+        _options = options;
+    }
+
+    public Meter CreateMeter(string name)
+    {
+        // TODO: Configure meter with default tags.
```

Comment:
>// TODO: Configure meter with default tags. [](http://example.com/codeflow?start=8&length=43)

I assume this will need to be supported by the Meter API. right?

Reply:
Yes